webui/react/package.json

```diff
@@ -137,9 +139,10 @@
     "morgan": "^1.10.0",
     "npm-force-resolutions": "0.0.10",
     "prettier": "^2.7.1",
+    "react": "^16.14.0",
```

Comment:
Is it an accident? 
I think `react` should be in dependencies and the version shouldn't go down